pyartcd/pyartcd/locks.py

```diff
@@ -1,49 +1,64 @@
+import enum
 import logging
 
 from aioredlock import Aioredlock
 
 from pyartcd import redis
 
+
+# Defines the pipeline locks managed by Redis
+class Lock(enum.Enum):
+    def __str__(self):
+        return f'{self.value}-lock'
+
+    OLM_BUNDLE = 'olm-bundle'
+    MIRRORING_RPMS = 'mirroring-rpms'
+    COMPOSE = 'compose'
+    GITHUB_ACTIVITY = 'github-activity'
```

Comment:
Can we keep these as `github-activity-{version}` ?
That way it's clear what we are expecting to go where in the string. And then the caller can use it by `.format(version=self.version)`

Reply:
Makes sense. I'll apply the same pattern everywhere else

Reply:
Done in https://github.com/openshift-eng/aos-cd-jobs/pull/3874/commits/2481f7fce81826afcf78ad994259dcfb54c9dac8